css/main.css

```diff
@@ -416,6 +421,62 @@ textarea {
     brightness(92%) contrast(82%);
 }
 
+@media (max-width: 720px) {
+  .menu-bars {
+    cursor: pointer;
+    z-index: 200;
+  }
+
+  #desktop-nav {
+    z-index: 100;
+    position: absolute;
```

Comment:
- [ ] The mobile menu appears when clicking the button (Really cool animation when appearing :fire:). But when scrolling down, it stays at the top of the page, and I think it would be better if it stays above the content even when scrolling. You could try using `position: fixed`, instead of absolute.

![image](https://user-images.githubusercontent.com/81522435/135772413-13567d96-6623-4975-a981-60c3e8ae3b52.png)

Reply:
- As the last reviewer suggest kindly change the position property to fixed instead of absolute in order to improve the visualization of your app.